### DISCUSSION TITLE
how can i create a private thread by using this libary???

### DISCUSSION THREAD

**Discussion by mansan1230** [Rapptz/discord.py#7711] [Q&A]
![image](https://user-images.githubusercontent.com/84826972/158951028-16645423-9462-4390-88db-e1b0929dbef2.png)
this libary only allow for create public thread

**Comment by Rapptz** [comment]
You use [`TextChannel.create_thread`](https://discordpy.readthedocs.io/en/master/api.html#discord.TextChannel.create_thread) instead.

**Reply by mansan1230** [reply]
this one is public thread, want private thread
how can I do ??

**Reply by drobotk** [reply]
It can create private threads too. Read the documentation carefully

**Reply by mansan1230** [reply]
![image](https://user-images.githubusercontent.com/84826972/158952021-43fd5ec1-57f2-4c8c-bf1c-fbd7e227f8c9.png)
my code there , and the library don't support other parameter

**Reply by bijij** [reply]
You're creating a thread from a message, not a channel, that's the distinction.
